packages/hawtio/src/ui/page/HawtioPage.tsx

```diff
@@ -72,7 +72,9 @@ export const HawtioPage: React.FunctionComponent = () => {
         defaultManagedSidebarIsOpen={showVerticalNavByDefault}
       >
         {/* Provider for handling selected node shared between the plugins */}
-        <PluginNodeSelectionContext.Provider value={{ selectedNode, setSelectedNode }}>
+        <PluginNodeSelectionContext.Provider
+          value={{ selectedNode, setSelectedNode }}
+        >
           <Routes>
```

Comment:
Cannot see why we are exposing the `setSelectedNode` to the plugins given the `HawtioPage` should be controlling the selection to ensure its common to all plugins.

Reply:
Ah ... now found the reason for `setSelectedNode`. Turns out the view trees control it too so my comment was invalid.
Am still unsure what to do about the `PluginNodeSelectionContext.Provider` being exposed outside of the `plugins` but works at the moment until we find something better.